src/_color-themes.scss

```diff
@@ -0,0 +1,103 @@
+// Themes
+
+$bg: "bg";
+$fg: "fg";
+$lightest-blue: "lightest-blue";
```

Comment:
We should think about refactoring all of these to a $colorNames map in variables.scss

Reply:
Another option would be to omit these. I only added them to be able to write `t($bg)` instead of `t("bg")`, so it's merely a matter of aesthetics which doesn't make a big difference.

Reply:
Yeah, I don't think it's necessary to have them as variables since they're only used in this file.